mappings/net/minecraft/item/Item.mapping

```diff
@@ -38,7 +39,7 @@ CLASS none/aeo net/minecraft/item/Item
 		ARG 0 tab
 	METHOD a getRawIdByItem (Lnone/aeo;)I
 		ARG 0 item
-	METHOD a (Lnone/aeo;Lnone/adq;Lnone/dc;)V
+	METHOD a addSubtypesToCreativeTab (Lnone/aeo;Lnone/adq;Lnone/dc;)V
```

Comment:
In Block we use "METHOD a addStacksForDisplay (Lnone/aeo;Lnone/adq;Lnone/dc;)V" and I think it makes more sense personally.